src/operator/make_loss.cc

```diff
@@ -37,7 +37,7 @@ The output of this function is the gradient of loss with respect to the input da
 For example, if you are a making a cross entropy loss function. Assume ``out`` is the
 predicted output and ``label`` is the true label, then the cross entropy can be defined as::
 
-  cross_entropy = label * log(out) + (1 - label) * log(1 - out)
+  cross_entropy = -mean(label * log(out) + (1 - label) * log(1 - out))
   loss = MakeLoss(cross_entropy)
```

Comment:
The normalization param of MakeLoss is set default to 'null', not 'batch'. I think that if the shapes of data and label are (N, C, H, W), the right way to calculate cross entropy, as I have tested, is to mean it on the (N, H, W) axes according to http://ufldl.stanford.edu/wiki/index.php/Softmax_Regression.